authortechnical.Rmd

```diff
@@ -253,6 +253,24 @@ knitr::include_graphics("images/imgtxt.png")
 <code>&lt;!\-\-/html_preserve\-\-></code>  
 
 
+#### Image gallery
```

Comment:
To match the heading style elsewhere? Or do you think they should all be `####`? Other than that, this looks great!
```suggestion
**Image gallery**
```